JavaAssignment/src/kunal/SumPrimeNumber.java

```diff
@@ -0,0 +1,34 @@
+package kunal;
+
+class SumPrimeNumber
+{
+	static boolean SumPrime(int num)
```

Comment:
Give proper method name, and incorporate rest comment, which i gave in other pull requests.